src/_posts/aframe-v0.3.0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: A-Frame v0.3.0 - Walk in a Web Page
+author: twitter|andgokevin|Kevin Ngo
+date: 2016-08-16 10:00:00
+layout: blog
+image:
+  src: v0.3.0-roomscale.gif
+---
+
+[webvr1.0]: https://w3c.github.io/webvr/
+
+A-Frame **v0.3.0** has dropped! *What is virtual reality without our legs nor our
+hands?* With support for the **[new WebVR 1.0 API][webvr1.0]**, we aim higher
+towards state of the art VR. With room-scale tracking, we can walk in a web
+page, and with tracked controller support, we can reach out into the world.
+
+<!-- more -->
+
+## What's New?
+
+**WebVR 1.0** support is the highlight of this release. This includes:
+
+- Better **room-scale** support to be able to walk/dash/duck/jump in a web
+  page with the HTC Vive.
+- Hitting **90 frames per second** due to being able to push content directly to the headset
+  display rather than mirroring a desktop display. This also allows us to have different
+  content on the desktop display than the headset, opening the door for asynchronous gameplay
+  or spectator modes.
+
+As a corollary, A-Frame now supports **tracked controllers** with the HTC Vive.
+Tracked controllers go hand-in-hand with room-scale:
+
+![A-Frame Ball Throw Demo](../../images/awoa/ball-throw.gif)
+
+[inspectordemo]: https://aframe.io/aframe-inspector/example/
+
+We are also releasing the much-anticipated **A-Frame Inspector** ([view
+demo][inspectordemo]). The current primary purpose of the Inspector is as a
+complementary tool for inspecting and tweaking scenes alongside code; it is
+more akin to the browser's DOM inspector rather than an end-to-end tool like
+Unity. The easiest way to try out the Inspector is to press `<ctrl> + <shift> +
+i` on any A-Frame scene using v0.3.0:
+
+![A-Frame Inspector Demo](../../images/blog/v0.3.0-inspector.gif)
+
+Other remarkable changes include large performance improvements, better support
+for webpage-embedded scenes, and support for multiple instances of components
```

Comment:
I would suggest maybe "multiple scenes embedded in a web page" instead of "webpage-embedded scenes." It's at least not clear to me what that term means (without having already known about what you're referring to).